receiver/mongodbreceiver/client_test.go

```diff
@@ -28,7 +28,6 @@ import (
 
 func TestListDatabaseNames(t *testing.T) {
 	mont := mtest.New(t, mtest.NewOptions().ClientType(mtest.Mock))
-	defer mont.Close()
```

Comment:
This cleanup now happens automatically: https://github.com/mongodb/mongo-go-driver/blob/v1/mongo/integration/mtest/mongotest.go#L184